ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1021,12 +1028,10 @@ def test_ipahealthcheck_trust_agent_member_check(self):
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] == self.master.hostname
 
-
 @pytest.fixture
 def modify_permissions():
     """Fixture to change owner, group and/or mode
-
-       This can run against multiple files at once but only one host.
+    This can run against multiple files at once but only one host.
```

Comment:
I got over eager when removing spaces, can you indent this +4 to line up with block?

Reply:
intended to +4 to lineup with block